### PR TITLE
Make ThreadDetails fill the whole height

### PR DESCRIPTION
**Pull request by elenaperezrioja**
## Summary

The thread on the ThreadDetails component was not filling the whole height when there is enough content. Now it is, by adding a bit of css and upgrading to 1.42.0

### CODE REVIEW

**Comment by elenaperezrioja** [comment]
* **#141** <a href="https://app.graphite.dev/github/pr/getcord/clack/141?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @elenaperezrioja and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/client/components/ThreadDetails.tsx [hunk 1]
```diff
@@ -333,11 +333,15 @@ const SeparatorLine = styled.hr`
...
   }
   &.cord-component .cord-composer {
     margin: 0 20px 20px;
   }
   &.cord-component .cord-thread-header-container {
     display: none;
   }
+  &.cord-component .cord-scroll-container {
+    max-height: none;
+  }
```
Comment (AlbertQM) [inline src/client/components/ThreadDetails.tsx:346]: Oh interesting, the `scroll-container` shouldn't have a `max-height` (it used to, but newer version don't) -- is it Clack setting it?

Comment (elenaperezrioja) [reply src/client/components/ThreadDetails.tsx:346]: Trying to find where it is coming from, but it doesn't look like it's coming from clack Screenshot 2024-04-24 at 14 00 17

Comment (AlbertQM) [reply src/client/components/ThreadDetails.tsx:346]: Oh, I see CSS version is 1.41.0 -- updating to 1.42 will fix it

**Review by AlbertQM — APPROVED** [review]
Thank you!

**Comment by elenaperezrioja** [comment]
I'll update then

**Pull request merged**